scripts/restore

```diff
@@ -20,14 +20,6 @@ ynh_restore_file --origin_path="$install_dir"
 chmod -R o-rwx "$install_dir"
 chown -R $app:www-data "$install_dir"
 
-#=================================================
-# REINSTALL DEPENDENCIES
-#=================================================
-ynh_script_progression --message="Reinstalling dependencies..." --weight=1
-
-# Define and install dependencies
-ynh_install_nodejs --nodejs_version=$nodejs_version
-
```

Comment:
Eeeeh pretty sure you do need this tho ? The restore is failing because the nodejs bin doesnt exist anymore on a fresh system

Reply:
Oh ok yeah I was confused because it's called a dependency I thought it would do a npm install